bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -1078,14 +1079,29 @@ private void checkStatus() {
             } else {
                 // Determine current status
                 var additionalConfiguration = AdditionalConfiguration.get(localRepo, localHash, pr.repository().forge().currentUser(), comments);
-                checkablePullRequest.executeChecks(localHash, censusInstance, visitor, additionalConfiguration);
+                checkablePullRequest.executeChecks(localHash, censusInstance, visitor, additionalConfiguration, true);
+                if (localRepo.isFileUpdated(".jcheck/conf")) {
```

Comment:
We probably don't need to run the second round if confOverride is set.

Reply:
Good point!